boards/arduino/portenta_h7/doc/index.rst

```diff
@@ -35,7 +35,7 @@ More information about STM32H747XIH6 can be found here:
 Supported Features
 ==================
 
-The current Zephyr arduino_portenta_h7 board configuration supports the following hardware features:
+The current Zephyr ``arduino_portenta_h7`` board target supports the following hardware features:
```

Comment:
and this is exactly as I mentioned, this does not exist, this needs to be `arduino_portenta_h7/stm32h747xx/m7`, if talking about the m7 core, or `arduino_portenta_h7/stm32h747xx/m4` if talking about the m4 core, you cannot just do a replacement with the folder name, that is not going to work for a huge number of cases

Reply:
> this does not exist, this needs to be arduino_portenta_h7/stm32h747xx/m7, if talking about the m7 core, or arduino_portenta_h7/stm32h747xx/m4 if talking about the m4 core

Thanks. 

@benjaminbjornsson ,  can you provide some input as the orginal author? N.b. there's no maintainers listed for this in `MAINTAINERS.yml`

> you cannot just do a replacement with the folder name, that is not going to work for a huge number of cases

This is using the output of `west boards` (which may or may not be just listing directory names). I don't have any numbers to quantify this, so I can't say what 'huge' means.

I'm willing to take a punt that the majority of in-tree boards have a single SoC, no board revision and/or the documentation. I feel like this commit doesn't make any document worse (if it was ambiguous before, then it still is). Perfect is the enemy of good, etc.